draft-iab-covid19-workshop.md

```diff
@@ -301,13 +303,14 @@ The impacts of this crisis are also a potential opportunity to
 understand the impact of traffic shifts and growth more generally, or
 to prepare for future situations — crises or otherwise – that impact
 networking. Or even allow us to adjust the technology to be even
-better suited to respond to changes.
+better suited to respond to changes in Internet traffic and user behavior.
 
 The scope of this workshop included:
 
-* measurements about traffic changes, user experience, service performance, and other relevant aspects
+* measurements about traffic changes, user experience and problems, service performance, and other relevant aspects
 * discussion about the behind the scenes network management and expansion activities
 * experiences in the fields of general Internet connectivity, conferencing, media/entertainment, and Internet infrastructure
+* the impact on networks of COVID-related scams and increased home working infrastructure, as threat actors made use of the attack surface resulting from the shift to home-working and a global pandemic
```

Comment:
Good to add another bullet point here but not sure I understand your wording: when you say "impact on networks", what networks do you mean? And what do you mean by "increased home working infrastructure"?

Reply:
```suggestion
* the impact of COVID-related scams on the home working infrastructure, as threat actors made use of the attack surface resulting from the shift to home-working in a global crisis
```